generators/ci-cd/templates/_.gitlab-ci.yml

```diff
@@ -1,21 +1,19 @@
-<%_ if (integrations.includes('docker')) { _%>
```

Comment:
Yes, it works. Now I know and can use. Maybe it's good idea just to add note about that to the documentation?